mysql-test/t/win_ntile.test

```diff
@@ -168,4 +168,51 @@ select pk, a, b,
 from t1;
 
 
+drop table t1;
+
+--echo #
+--echo # MDEV-9911 NTILE must return an error when parameter is not stable 
```

Comment:
Here is whitespace at the end, no need.
Please take a look of other white space cases.
You can set your editor to show you whitespace changes when writing a patch.